pkg/cloudprovider/aws/fleet/packing/podresources.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Can we pull these helpers out of the cloud provider? Into util/binpacking maybe? binpacking.SortablePods